mllib/src/test/java/org/apache/spark/ml/feature/JavaBucketizerSuite.java

```diff
@@ -17,7 +17,8 @@
 
 package org.apache.spark.ml.feature;
 
-import com.google.common.collect.Lists;
+import java.util.Arrays;
```

Comment:
What is the correct import order for java? I followed the scala import guidelines here